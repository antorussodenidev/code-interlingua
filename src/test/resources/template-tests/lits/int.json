{
    "type": {
        "nodeType": "Type",
        "code": "int"
    },
    "value": 42
}
