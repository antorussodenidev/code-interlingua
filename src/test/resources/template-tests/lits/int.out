package foo.bar;

class C {
  static final int X = 42;

  public C() {}
}
